### PR TITLE
Features/cit 309/timestamp fixedoffset

### PR DESCRIPTION
**Pull request by PradeepKiruvale**
This PR  addresses the issue of timestamp parameter in thinedge struct from string to datetime type

### CODE REVIEW

File: mapper/cumulocity/c8y_translator_lib/src/lib.rs [hunk 1]
```diff
@@ -172,14 +179,14 @@ mod tests {
     #[test]
     fn check_thin_edge_translation_with_timestamp() {
         let single_value_thin_edge_json = r#"{
-                  "time" : "2013-06-22T17:03:14.000+02:00",
+                  "time" : "2013-06-22T17:03:14+02:00",
```
Comment (didier-wenzek) [inline mapper/cumulocity/c8y_translator_lib/src/lib.rs:182]: The test expectations MUST not be changed during a refactoring.

The timestamp should be printed with a millisecond precision.

**Review by didier-wenzek — CHANGES_REQUESTED** [review]
* The timestamp should be printed with a millisecond precision as before the refactoring.
* Add a test checking that millisecond are preserved.

File: mapper/cumulocity/c8y_translator_lib/src/lib.rs (hunk 1 above)
Comment (PradeepKiruvale) [reply mapper/cumulocity/c8y_translator_lib/src/lib.rs:182]: Agree, but when I convert the RC3339 format timestamp string like here, DateTime::parse_from_rfc3339(&timestamp_str), strips of the milliseconds for some reason. I do not see any other API that can help here. Either we should parse string ourself of milliseconds and add later to fixedoffset time stamp.
Let me know if there any API that can help here.

Comment (didier-wenzek) [reply mapper/cumulocity/c8y_translator_lib/src/lib.rs]: So update the test with non zero milli-seconds to make this obvious.

**Review by didier-wenzek — COMMENTED** [review]
I will approve - once the git log fixed.

**Comment by didier-wenzek** [comment]
Has been replaced by https://github.com/thin-edge/thin-edge.io/pull/174

**Pull request closed without merging**